NuRadioReco/modules/trigger/highLowThreshold.py

```diff
@@ -136,10 +142,27 @@ def run(self, evt, station, det,
             a unique name of this particular trigger
         set_not_triggered: bool (default: False)
             if True not trigger simulation will be performed and this trigger will be set to not_triggered
-
+        Vrms: float
+            If supplied, overrides adc_reference_voltage as supplied in the detector description file
+        trigger_adc: bool
+            If True, the relevant ADC parameters in the config file are the ones
+            that start with `'trigger_'`
+        clock_offset: float
+
+        adc_output: string
+            Options:
+            * 'voltage' to store the ADC output as discretised voltage trace
+            * 'counts' to store the ADC output in ADC counts
+
+        use_digitization: bool
+            If True, traces will be digitized
```

Comment:
Sorry to be picky, but I think just to be more helpful to users we should reorder the options and set `use_digitization` first and then add a "(Only active if `use_digitization` is set to `True`)" for the other options.

Reply:
I've just reordered them and added a comment right next to the `use_digitization` argument.